nestle.py

```diff
@@ -583,6 +651,13 @@ def update(self, pointvol):
         self.ell = bounding_ellipsoid(self.points, pointvol=pointvol)
         self.ell.scale_to_vol(pointvol)
 
+    def propose_point(self, u):
+        while True:
+            new_u = u + scale * self.ell.randoffset(rstate=self.rstate)
```

Comment:
@kbarbary Not as far as I can tell, since this particular `propose_point` (and its use of `scale`) is already specific to the MCMC implementation.
@aarchiba I don't think the algorithm is "wrong", although it depends on what you're thinking of when you run the `classic` sampler. Ultimately, MCMC-like proposals are just a way to random-walk away from an original trial location to get an independent sample. As implemented, the proposal distribution changes over the course of a call to `new_point` to achieve a target acceptance ratio of 0.5. This violates detailed balance, but doesn't necessarily mean it's not an effective random walk. I haven't done any tests to see whether "fixing" this issue would improve things though.